config/routes.rb

```diff
@@ -19,6 +19,7 @@
         resources :passwords, only: :create
       end
 
+      resources :graphql, only: :create
```

Comment:
Not sure about the route location as 1) `api/v1` is meant only for the REST API and 2) GraphQL does not handle versioning with paths but with deprecated fields. Having simply the endpoint `/graphql` would do as done similarly by [Github](https://developer.github.com/v4/guides/forming-calls/#the-graphql-endpoint).

Reply:
Fixed in https://github.com/nimblehq/nimble-survey-web/pull/27/commits/1f8c00fe60f1730685034b078441870311349a29